spec/models/course/video_spec.rb

```diff
@@ -58,6 +58,26 @@
       end
     end
 
+    describe '.next_video' do
```

Comment:
Sorry to be pedantic, but this should be `#next_video` because it's an instance method.

Reply:
Ic, updated